base/markdown/GitHub/table.jl

```diff
@@ -78,7 +78,7 @@ end
 mapmap(f, xss) = map(xs->map(f, xs), xss)
 
 colwidths(rows; len = length, min = 0) =
-    reduce(max, [min; convert(Vector{Vector{Int}}, mapmap(len, rows))])
+    reduce((x,y) -> max.(x,y), [min; convert(Vector{Vector{Int}}, mapmap(len, rows))])
```

Comment:
My thought as well. But that replacement caused test failures (see the original post's checklist's last item). I arrived at the present solution after examining that line more thoroughly. The details escape me due to the intervening time.